.github/config.yml

```diff
@@ -0,0 +1,27 @@
+# https://github.com/behaviorbot/welcome
+# combines new-issue-welcome, new-pr-welcome, and first-pr-merge
+
+# Comment on first-time issues
+# https://github.com/behaviorbot/new-issue-welcome
+newIssueWelcomeComment: >
+  👋 Thanks for opening your first issue here! If you're reporting a bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+
+  To make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/issues.md).
+
+# Comment on PRs from first-time contributors
+# https://github.com/behaviorbot/new-pr-welcome
+newPRWelcomeComment: >
+  💚 Thanks for opening this pull request! 💚
+
+  Please make sure to update documentation and tests when appropriate, and follow the [commit guidelines](https://github.com/nodejs/node/blob/master/doc/guides/contributing/pull-requests.md#commit-message-guidelines)
+
+  For non-trivial changes, pull requests must be left open for *at least* 48 hours during the week, and 72 hours on a weekend.
+
+  If this pull request changes includes code changes, the CI must be started by a collaborator.
```

Comment:
Can you link `collaborator` to https://github.com/nodejs/node/blob/master/GOVERNANCE.md#collaborators ?